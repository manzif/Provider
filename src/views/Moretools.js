import React, { Component } from "react";

export class Moretools extends Component {
  
  render() {
      return (
        <>
        <div className="popup">
            <a href="/" className="popup__close">&times;</a>
                <div className="popup__tools">
                    
                </div>
            </div>
        </>
      );
    }
  }

export default Moretools
