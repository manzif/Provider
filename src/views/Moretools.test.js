import React from "react";
import { shallow } from "enzyme";
import { Moretools } from "./Moretools";

describe("landing page", () => {
  const component = shallow(<Moretools  />);
  it("renders without errors", () => {
    expect(component).toMatchSnapshot();
  });
});
