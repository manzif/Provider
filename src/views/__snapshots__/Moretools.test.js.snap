// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing page renders without errors 1`] = `ShallowWrapper {}`;
